plugins/generic/acron/locale/fr_CA/locale.xml

```diff
@@ -14,7 +14,11 @@
  
 <locale name="fr_CA" full_name="Français (Canada)">
 	<message key="plugins.generic.acron.name">Plugiciel Acron</message>
+	<message key="plugins.generic.acron.description">Ce plugiciel tente de réduire la dépendance d'OCS aux outils de planificiation périodique tel que 'cron.'</message>
```

Comment:
This is not the most recent version of that key, please double-check the en_US version.